src/views/Room.vue

```diff
@@ -62,10 +65,12 @@
     import Settings from '../components/Settings';
     import ChatMessageNotification from '../components/ChatMessageNotification';
     import { uniqBy } from 'lodash/array';
+    import InviteUsers from '@/components/InviteUsers';
```

Comment:
Change to ../components instead of @/components, or change the other imports

this could lead to confusion down the road